react/src/views/Box/BoxView.tsx

```diff
@@ -333,7 +368,21 @@ function BTBox() {
         newLocationId: parseInt(locationId, 10),
       },
       refetchQueries: [refetchBoxByLabelIdentifierQueryConfig(labelIdentifier)],
-    });
+    })
+      .then((res) => {
+        notificationVar({
+          title: `Box ${labelIdentifier}`,
+          type: res.errors ? "error" : "success",
+          message: res.errors ? "Error while trying to move the box" : "Successfully moved the box",
+        });
+      })
+      .catch(() => {
+        notificationVar({
+          title: `Box ${labelIdentifier}`,
+          type: "error",
+          message: "Error while trying to move the box",
```

Comment:
"Error: Box could not be moved!" should be the error. Then what happens? Does the whole action fail? Should the user try again?